app/js/daemonManager.js

```diff
@@ -61,6 +61,9 @@ function DaemonManager() {
 		});
 	}
 
+	/**
+	 * Checks if there is an update available
```

Comment:
I like the go strategy of including the function name in the function docstring

updatePrompt checks if there is an update available

sometimes it helps when grepping